docs/commands/GetArchiveEvents.md

```diff
@@ -0,0 +1,105 @@
+# GetArchiveEvents
```

Comment:
would be nice to add new commands to the command list

Reply:
added